kubernetes/omsagent.yaml

```diff
@@ -379,13 +379,19 @@ spec:
               memory: 225Mi
           env:
             # azure devops pipeline uses AKS_RESOURCE_ID and AKS_REGION hence ensure to uncomment these
+            - name: AKS_CLUSTER_NAME
+              value: "VALUE_AKS_CLUSTER_NAME"
             - name: AKS_RESOURCE_ID
               value: "VALUE_AKS_RESOURCE_ID_VALUE"
             - name: AKS_REGION
               value: "VALUE_AKS_RESOURCE_REGION_VALUE"
+            - name: FBIT_SERVICE_FLUSH_INTERVAL
```

Comment:
wouldn't it make sense to ad all the settings here and make then the default value ? But then i am assuming if customer over-rides them with a configmap, that will win over the defaults ?

Reply:
Since we have reverted to default FBIT settings - https://github.com/microsoft/Docker-Provider/commit/cd2275354aafb588f0ed74cf8d747e40226e4974 and this will be removed, and there is a task on July release story to remove this from AKS RP yamls.

Reply:
Removed this, since we will be reverting.